### ISSUE TITLE
Nothing on stdout

### ISSUE THREAD

**Issue by karel-3d** [dcousens/fast-dat-parser#8]
I might be missing something

When I try to run the parser, I see messages on stderr, but nothing in stdout

````
cat ~/.bitcoin/blocks/blk00000.dat | ./parser -j4 -t1 > wat
Allocated IO buffer (104857600 bytes)
Allocated parse buffer (104857600 bytes)
Initialized 4 threads in the thread pool
-- Parsed 0 blocks (read 102400 KiB, 100 MiB total, skipped 0KiB)
-- Parsed 114484 blocks (read 28670 KiB, 127 MiB total, skipped 0KiB) EOF
Parsed 119969 blocks (127 MiB)in 1 seconds
````

but
````
$ wc -l wat
0
````

only `-t0` returns binary output. Am I running it wrong?

**Comment by runn1ng** [comment]
Oh nevermind, now I understand I first have to make the whitelist. Closing

**Comment by dcousens** [comment]
@runn1ng odd,  [if the the whitelist was empty,  it should output every block](https://github.com/dcousens/fast-dat-parser/blob/master/src/transforms.hpp#L46),  did adding `-w` really resolve your issue?

**Comment by dcousens** [comment]
Resolved in c110b97
